File: client/src/main/java/com/sio/pizzeria/DTO/CommandePayerDTO.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package com.sio.pizzeria.DTO;

import java.util.Date;

/**
 *
 * @author Lenovo P50
 */
public class CommandePayerDTO {
    

    private int idCommandePayer;
    private int idHistoriqueProduit;
    private int idHistoriqueTable;
    private int nbProduit;
    private Date date;

    public CommandePayerDTO(int idHistoriqueProduit, int idHistoriqueTable, int nbProduit, Date date) {
        this.idCommandePayer = idCommandePayer;
        this.idHistoriqueProduit = idHistoriqueProduit;
        this.idHistoriqueTable = idHistoriqueTable;
        this.nbProduit = nbProduit;
        this.date = date;
    }

    
    
    public int getIdCommandePayer() {
        return idCommandePayer;
    }

    public void setIdCommandePayer(int idCommandePayer) {
        this.idCommandePayer = idCommandePayer;
    }

    public int getIdHistoriqueProduit() {
        return idHistoriqueProduit;
    }

    public void setIdHistoriqueProduit(int idHistoriqueProduit) {
        this.idHistoriqueProduit = idHistoriqueProduit;
    }

    public int getIdHistoriqueTable() {
        return idHistoriqueTable;
    }

    public void setIdHistoriqueTable(int idHistoriqueTable) {
        this.idHistoriqueTable = idHistoriqueTable;
    }

    public int getNbProduit() {
        return nbProduit;
    }

    public void setNbProduit(int nbProduit) {
        this.nbProduit = nbProduit;
    }

    public Date getDate() {
        return date;
    }

    public void setDate(Date date) {
        this.date = date;
    }
    
    
}

File: server/DTO/CommandePayerDTO.php
<?php


class CommandePayerDTO implements JsonSerializable
{
    private $idCommandePayer;
    private $idHistoriqueProduit;
    private $idHistoriqueTable;
    private $nbProduit;
    private $date;

    public function __construct()
    {
    }


    public function getIdCommandePayer()
    {
        return $this->idCommandePayer;
    }

    public function setIdCommandePayer($idCommandePayer)
    {
        $this->idCommandePayer = $idCommandePayer;
    }

    public function getIdHistoriqueProduit()
    {
        return $this->idHistoriqueProduit;
    }

    public function setIdHistoriqueProduit($idHistoriqueProduit)
    {
        $this->idHistoriqueProduit = $idHistoriqueProduit;
    }

    public function getIdHistoriqueTable()
    {
        return $this->idHistoriqueTable;
    }

    public function setIdHistoriqueTable($idHistoriqueTable)
    {
        $this->idHistoriqueTable = $idHistoriqueTable;
    }

    public function getNbProduit()
    {
        return $this->nbProduit;
    }

    public function setNbProduit($nbProduit)
    {
        $this->nbProduit = $nbProduit;
    }

    public function getDate()
    {
        return $this->date;
    }

    public function setDate($date)
    {
        $this->date = $date;
    }

    public function jsonSerialize()
    {
        return array(
            'idCommandePayer' => $this->idCommandePayer,
            'idHistoriqueProduit' => $this->idHistoriqueProduit,
            'idHistoriqueTable' => $this->idHistoriqueTable,
            'nbProduit' => $this->nbProduit,
            'date' => $this->date,
        );
    }

}